meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -58,6 +60,16 @@ export interface MongoPart {
 	title: string
 	_rank: number
 	expectedDuration: number
+	inTransition?: {
+		previousPartKeepaliveDuration: number
```

Comment:
I would say that `previousPartKeepaliveDuration` should be `previousPartKeepAliveDuration`

Reply:
I agree. Unfortunately this is how the column is named in the database and the purpose of this interface is to mimic the database so we have an "easy" conversion.
Notice that it maps it to `Part.keepPreviousPartAliveDuration`